sdk/spring-sdk/src/it/java/com/example/wiring/SpringSdkWiringIntegrationTest.java

```diff
@@ -102,4 +114,75 @@ public void verifyCounterEventSourcedWiring() {
 
     Assertions.assertEquals("\"200\"", counterGet);
   }
+
+  @Test
+  public void verifyFindUsersByEmail() {
+
+    ResponseEntity<String> response =
+        webClient
+            .post()
+            .uri("/user/jane/jane/jane.example.com")
```

Comment:
I think this is why the test is failing

```suggestion
            .uri("/user/jane/jane.example.com/jane")
```